install/linux/docker-ce/ubuntu.md

```diff
@@ -68,36 +67,11 @@ outlined below.
 
```

Comment:
In the section above, we can probably change the bullet list (because all supported Ubuntu versions are now on kernel 4 or above;

```Markdown
 - For new installations on version 4 and higher of the Linux kernel, `overlay2`
   is supported and preferred over `aufs`.
 - For version 3 of the Linux kernel, `aufs` is supported because `overlay` or
   `overlay2` drivers are not supported by that kernel version.
```

Perhaps combine the first bullet with the first sentence; something like;

```Markdown
Docker CE on Ubuntu supports `overlay2` and `aufs` storage drivers. For new
installations, `overlay2` is the default, and preferred over `aufs`. If you
need to use `aufs`, you need to do additional preparation as outlined below.
```